testdata/baselines/reference/submodule/conformance/callbackTagVariadicType.js.diff

```diff
@@ -23,7 +23,7 @@
 
 
  //// [callbackTagVariadicType.d.ts]
-+export type Foo = (args: string[]) => number;
++export type Foo = (...args: string) => number;
```

Comment:
Nothing you did in this PR, but this looks like a declaration emit bug where `callback` is not supposed to be exported, right?

Reply:
well, no, `@callback` is like `@typedef` in that it's implicitly exported.

Reply:
Except the original d.ts file didn't do so, no?

Reply:
Oh, I can't read, it's just an ordering problem, I can see it below hidden in the github diff.